boards/arduino-due/include/periph_conf.h

```diff
@@ -342,6 +343,34 @@ extern "C" {
 #define GPIO_B21_MAP        31
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define MCK_DIV_LB_MAX      (10U)
```

Comment:
Can you explain this value? Is there maybe a more self-explaining name? I'm wondering if it's reasonable to define this macro in the periph_conf